modules/serverless-jaeger-config.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * /serverless/serverless-tracing.adoc
+
+[id="serverless-jaeger-config_{context}"]
+= Configuring Jaeger for use with {ServerlessProductName}
+
+.Procedure
+
+. Create and apply the Jaeger custom resource:
++
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+----
+
+// TODO: Add images to show configuring this
+. Enable tracing on for Knative Serving:
++
+----
+$ oc edit ks -n knative-serving knative-serving
+----
++
+.Example YAML
+----
+tracing:
+  sample-rate: "0.1"
+  backend: zipkin
+  zipkin-endpoint: http://jaeger-collector.<JAEGER_NAMESPACE>.svc.cluster.local:9411/api/v2/spans
+  debug: "true"
+----
++
+* The backend must be `zipkin`.
+// why? Check Jaeger docs.
```

Comment:
It is a Knative's specific configuration. `zipikin` is has same tracing API and we can use it via Jaeger.

It is strange, but it is correct :smile_cat: